avm/res/net-app/net-app-account/tests/e2e/max/dependencies.bicep

```diff
@@ -1,5 +1,6 @@
-@description('Optional. The location to deploy to.')
-param location string = resourceGroup().location
+// enforcing location due to quote restrictions
+#disable-next-line no-hardcoded-location
+var enforcedLocation = 'australiaeast'
```

Comment:
No need to hardcode this here, the enforced location is passed in from outside and overwritte. In other words, the change in the `dependencies.bicep` file can be undone 😉